pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -284,7 +321,7 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication {
      * @dev Sets the root address to `newRoot`.
      */
     function setRoot(address newRoot) external onlyExecutor {
```

Comment:
Just throwing out there whether this is important enough to justify a claim step, vs just having the timelock. setRoot would then just set _rootCandidate, and the new root account would have to call claimRoot(), which would set _root and emit the event.

Reply:
Yeah, I'd agree that this is something we want to have as many failsafes as possible on. I'm made an issue to track this in #1312 but I think it falls outside the scope of this PR.